queue/abstract/driver/fifo.lua

```diff
@@ -33,8 +33,21 @@ function tube.create_space(space_name, opts)
     return space
 end
 
+-- validate space of queue
+local function validate_space(space)
```

Comment:
I agree, WET practice can be used here.
But I suggest using something like:
"
local function validate_space(space, indexes)
	local indexes = {'task_id', 'status'}
	for _, index in pairs(indexes) do
		if space.index[index] == nil then
			error('space does not have ' .. index .. ' index')
		end
	end
"